server/README.md

```diff
@@ -0,0 +1,321 @@
+
+To do this, cd into the BraveSensor-Server directory and run the following command
+
+```
+sudo PG_PORT=<your db's port> PG_HOST=<your db's host> PG_PASSWORD=<your db's password> PG_USER=<your db's user> PG_DATABASE=<your db name> ./setup_postgresql_local.sh
+```
+
+## Tests
+
+Unit tests are written using the [Mocha](https://mochajs.org/) JS test framework, the [Chai](https://www.chaijs.com/) assertion library, and the [Sinon](https://sinonjs.org/) mocking library.
+
+To run the tests locally:
+
+```
+npm install
+npm run test
+```
+
+The tests run automatically on Travis on every push to GitHub and every time a pull request is created.
+
+## How to add or change an encrypted Travis environment variable
+
+Reference: https://docs.travis-ci.com/user/environment-variables/#encrypting-environment-variables
+
+1. Download the Travis CLI `gem install travis`
+
+1. cd to anywhere in this repo
+
+1. temporarily create a personal access token on GitHub https://github.com/settings/tokens with the following permissions:
+
+   - `repo`
+   - `read:packages`
+   - `read:org`
+   - `read:public_key`
+   - `read:repo_hook`
+   - `user`
+   - `read:discussion`
+   - `read:enterprise`
+
+1. login using `travis login --pro --github-token <token from github>`
+
+1. For a given `VAR_NAME` that you want to have value `secret_value`, run
+   `travis encrypt --pro VAR_NAME=secret_value`
+   which will ask for your GitHub username and password and then
+   output your encrypted variable
+
+1. Copy the encrypted variable into `.travis.yml`
+
+1. Delete your personal access token from GitHub
+
+# Sensors Admin Server
+
+The Sensors Admin server is used for **dev**, **staging**, and **prod**.
+
+To ssh onto the `sensors-admin` server
+
+```
+ssh brave@sensors-admin.brave.coop
+```
+
+The `sudo` password is on 1Password --> ODetect Credentials --> Login - sensors-admin
+(cluster management) server.
+
+## Adding public keys to the admin server
+
+Access is restricted to machines whose public key has been added to the server's allowlist.
+
+1. Using an account that can, SSH onto the `sensors-admin` server
+
+1. Paste the user's public key onto a new line on the `~/.ssh/authorized_keys` file from a machine that is already capable of accessing the server
+
+# Interacting with the Managed Database
+
+## Connecting to a database
+
+The PostgreSQL DB connection parameters are required for the application to open a connection to the database. We primarily use the following databases, although you are free to create new ones for development purposes
+
+| Environment | User      | Database Name       |
+| ----------- | --------- | ------------------- |
+| Production  | doadmin   | backend-replacement |
+| Staging     | bravetest | staging             |
+| Development | bravetest | bravetest           |
+
+To access a database shell
+
+1. Log in to Digital Ocean
+
+1. Navigate to Databases --> odetect-db
```

Comment:
Oh, the DB name can be updated to `odetect-db-jul-22-backup` while you're in here.